src/app/tests/suites/commands/commissioner/CommissionerCommands.cpp

```diff
@@ -33,6 +33,8 @@ CommissionerCommands::PairWithCode(const char * identity,
     memcpy(code, value.payload.data(), value.payload.size());
     ChipLogError(chipTool, "Pairing Code is %s", code);
 
+    mDiscoverOnce.SetValue(value.discoverOnce.ValueOr(false));
```

Comment:
Is there a reason this is not just:
```suggestion
    mDiscoverOnce = value.discoverOnce;
```
?